### ISSUE TITLE
Missing data in config files

### ISSUE THREAD

**Issue by paneil** [Skidamek/AutoModpack#137]
### Bug description

The client is missing a few bits of data and sometimes duplicating it in the config files which is causing errors. It seems like AutoModpack is not fully downloading the files from the modpack correctly. All the config files in the modpack is correct.

Client lambdynlights.toml content:
```
ight_sources.settings.entities.goodall]
				songbird = true
```
Modpack lambdynlights.toml content:
```
			[light_sources.settings.entities.goodall]
				songbird = true
```

Client dashloader.json content:
```
  "customSplashLines": [],
  "addDefaultSplashLines": true
}nna let you down", "Never gonna run around and desert you", "Never gonna make you cry", "Never gonna say goodbye", "Never gonna tell a lie and hurt you"],
  "addDefaultSplashLines": true
}dDefaultSplashLines": true
}
```
Modpack dashloader.json content:
```
"customSplashLines": ["Never gonna give you up", "Never gonna let you down", "Never gonna run around and desert you", "Never gonna make you cry", "Never gonna say goodbye", "Never gonna tell a lie and hurt you"],
  "addDefaultSplashLines": true
}
```

### Steps to reproduce

1. Create a modpack with mod config files
2. Client downloads the modpack
3. Compare the modpack's config files and modpack's config files.

### Expected behavior

The config file is a 1:1 copy

### Actual behavior

The client's config file contains duplicated and missing data which is different to the modpack's config file

### Minecraft version & Mod Loader

1.19.2 fabric

### Operating system

Windows 11

### AutoModpack version

3.3.8

### Other information

This bug originates from 3.3.7

### Check list

- [X] I have verified that the issue persists in the latest version of the mod.
- [X] I have searched the existing issues and confirmed that this is not a duplicate.

**Comment by Skidamek** [comment]
Huh this is so strange, could you send client log when you downloading whole modpack and the modpack-content.json file from server's /automodpack/host-modpack/ dir?

**Comment by Skidamek** [comment]
Could you send few examples of *wrongly* downloaded files and correct files?

**Comment by Skidamek** [comment]
I just can't reproduce that, everything downloads correctly

**Comment by Skidamek** [comment]
[automodpack-mc1.19.2-fabric-3.3.9-SHAPSHOT.zip](https://github.com/Skidamek/AutoModpack/files/11913670/automodpack-mc1.19.2-fabric-3.3.9-SHAPSHOT.zip)
Could you try this build?

**Comment by Skidamek** [comment]
Great, i changed it a bit again could you try this build? 
[automodpack-mc1.19.2-fabric-3.4.0+build.67.zip](https://github.com/Skidamek/AutoModpack/files/12113565/automodpack-mc1.19.2-fabric-3.4.0%2Bbuild.67.zip)

**Comment by Skidamek** [comment]
Fixed in 3.4.0
